Content.Shared/Clumsy/ClumsySystem.cs

```diff
@@ -40,6 +40,10 @@ private void BeforeHyposprayEvent(Entity<ClumsyComponent> ent, ref SelfBeforeHyp
         // Clumsy people sometimes inject themselves! Apparently syringes are clumsy proof...
         if (!_random.Prob(ent.Comp.ClumsyDefaultCheck))
             return;
+		
+		// imp. checks if ClumsyHypo is false, if so, skips.
+		if (!ent.Comp.ClumsyHypo)
```

Comment:
Put this above the randomization so we don't have to do that in case it's disabled. Same for the other new bools.

Reply:
done.